remotespark/utils/sparkevents.py

```diff
@@ -0,0 +1,32 @@
+import remotespark.utils.configuration as conf
+
+
+class SparkEvents:
+    def __init__(self):
+        """
+        Create an instance from the handler mentioned in the config file.
+        """
+        events_handler = self._import_handler(conf.events_config()["handler"])
```

Comment:
There is no reason for `events_config` to be a dictionary with a `"handler"` element -- just make it a string